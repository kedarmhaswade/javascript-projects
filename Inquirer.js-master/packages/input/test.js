const Input = require('.');

describe('Input prompt', () => {
  it('renders in idle state', () => {
    const output = Input.render({
      prefix: '?',
      message: 'Question:',
      value: 'answer',
      status: 'idle'
    });
    expect(output).toMatchSnapshot();
  });

  it('renders in done state', () => {
    const output = Input.render({
      prefix: '?',
      message: 'Question:',
      value: 'answer',
      status: 'done'
    });
    expect(output).toMatchSnapshot();
  });
});
