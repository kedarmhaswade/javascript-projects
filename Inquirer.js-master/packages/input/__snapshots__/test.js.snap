// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input prompt renders in done state 1`] = `"? Question: [36manswer[39m"`;

exports[`Input prompt renders in idle state 1`] = `"? Question: answer"`;
